CHANGELOG.md

```diff
@@ -1,12 +1,80 @@
 # Changelog
 
-## Unreleased
+## v6.0.0-rc.2 - 2017-11-14
 
-- Update the contribution guideline to note that updating the changelog is required. (#5586)
+### Breaking changes
+
+- Skip null-element check for collections with a value-type element type (#5432)
+- Make 'AllSigned' execution policy require modules under $PSHome to be signed (#5511)
+
+### Engine updates and fixes
+
+- Update powershell to use `2.0.4` dotnet core runtime. (#5677)
+- Remove references to the old executable `powershell` or `powershell.exe` (#5408)
+
+### General cmdlet updates and fixes
+
+- Remove unncessary check for Paths.count > 0, in the `*-FileCatalog` CmdLets, as there is code later to use the current (#5596)
+- Use explicit libpsl-native binary name for dllimport. (#5580)
+
+### Build and Packaging Improvements
+
+- Fix Get-EnvironmentInformation to properly check for CoreCLR (#5592) (Thanks @markekraus!)
+- Make Travis CI use `libcurl+openssl+gssapi` (#5629) (Thanks @markekraus!)
+- Disambiguate icon for daily builds on Windows (#5467) (Thanks @bergmeister!)
+- Fix Import-CliXml tests which still use `powershell` instead of `pwsh` and make sure it fails if it regresses (#5521) (Thanks @markekraus!)
+- Update port used to WebCmdLet test which broke due to a change in AppVeyor (#5520) (Thanks @markekraus!)
+- Clean up use of 'Runspaceconfiguration' from comments and xunit test code (#5569) (Thanks @Bhaal22)
 - Remove Pester as a module include with the PowerShell Packages.
   In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
```

Comment:
Remove `In the future`, they should be able to do this today for their own tests.

Reply:
resolved